pkg/messaging/rabbitmq/pubsub.go

```diff
@@ -34,6 +34,7 @@ var (
 	errAlreadySubscribed = errors.New("already subscribed to topic")
 	errNotSubscribed     = errors.New("not subscribed")
 	errEmptyTopic        = errors.New("empty topic")
+	errEmptyID           = errors.New("empty ID")
```

Comment:
If I remember well we are not using upper-case in logs

Reply:
This is also linked with the RabbitMQ branch let me update it [here](https://github.com/mainflux/mainflux/pull/1570)